tools/cli/build.gradle.kts

```diff
@@ -17,17 +17,6 @@ application {
     version = versionName
 }
 
-java {
-    sourceCompatibility = JavaVersion.VERSION_21
-    targetCompatibility = JavaVersion.VERSION_21
-}
-
-kotlin {
-    compilerOptions {
-        jvmTarget = JvmTarget.JVM_21
```

Comment:
Fallback to the default, Java 17. Seems no need to require it running on Java 21.